java/ql/src/utils/FlowTestCaseSupportMethods.qll

```diff
@@ -55,19 +55,25 @@ private Content getContent(SummaryComponent component) {
 
 module SupportMethod {
   GenMethod genMethodForContent(SummaryComponentStack c) {
-    result = min(GenMethod g | g.appliesTo(any(VoidType v), getContent(c.head())))
+    result = genMethodFor(any(VoidType v), c)
   }
 
   GenMethod genMethodFor(Type t, SummaryComponentStack c) {
-    result = min(GenMethod g | g.appliesTo(t, getContent(c.head())))
+    result =
+      min(GenMethod g |
```

Comment:
Looks like you can order by more than one thing in a single `min` expression -- for example, `min(MyClass c | ... | c order by c.order1(), c.order2())`